src/lib/Engineer.js

```diff
@@ -0,0 +1,19 @@
+const Employee = require("./Employee");
+
+class Engineer extends Employee {
+  constructor(name, id, email, github) {
```

Comment:
As commented in your other file, we can prep these classes to expect objects containing the required keys, rather than 4 separated arguments. Here we would see:

``` 
constructor( { name, id, email, github } ) { 
    super( { name, id, email } ) 
......
```

See other comment for a detailed explanation of why!